imagelib/search.php

```diff
@@ -45,7 +45,7 @@
 	<title><?php echo $DEFAULT_TITLE; ?> Image Library</title>
 	<meta name='keywords' content='Search, Images, Taxon' />
 	<?php
-	include_once($SERVER_ROOT.'/includes/head.php');
+	include_once($SERVER_ROOT.'/includes/head_template.php');
```

Comment:
head.php is created upon run of Symbiota/config/setup.sh when a Symbiota instance is first being created.

head.php is not tracked in git; it allows different instances of Symbiota to have different styling files and styling generally. We should revert back to head.php; it is safe to assume that this file will exist for all production instances of Symbiota.